tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -48,7 +57,66 @@
     fake_beard_api_request,
     fake_magpie_api_request,
 )
-from utils import get_halted_workflow
+from utils import (
+    _delete_record,
+    get_halted_workflow,
+)
+
+
+@pytest.fixture(scope='function')
+def workflow(workflow_app):
+    workflow_object = workflow_object_class.create(
+        data={},
+        id_user=1,
+        data_type="hep"
+    )
+    workflow_object.save()
+    db.session.commit()
+    workflow_object.continue_workflow = lambda **args: True
+
+    yield workflow_object
+
+    WorkflowObjectModel.query.filter_by(id=workflow_object.id).delete()
+    db.session.commit()
+
+
+@pytest.fixture(scope='function')
+def insert_journals_in_db(workflow_app):
+    """Temporarily add few journals in the DB"""
+    from inspirehep.modules.migrator.tasks import record_insert_or_replace  # imported here because it is a Celery task
+
+    journal_no_pro_and_ref = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'jou_record_refereed.json')))
+
+    journal_pro_and_ref = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'jou_record_refereed_and_proceedings.json')))
+
+    journal_no_pro_and_not_ref = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'jou_record_not_refereed_and_no_proceedings.json')))
+
+    journal_pro_and_no_ref_a = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'jou_record_not_refereed_and_proceedings.json')))
+
+    journal_pro_and_no_ref_b = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'jou_record_not_refereed_and_proceedings_2.json')))
+
+    with db.session.begin_nested():
+        record_insert_or_replace(journal_no_pro_and_ref)
+        record_insert_or_replace(journal_pro_and_ref)
+        record_insert_or_replace(journal_no_pro_and_not_ref)
+        record_insert_or_replace(journal_pro_and_no_ref_a)
+        record_insert_or_replace(journal_pro_and_no_ref_b)
+    db.session.commit()
+    es.indices.refresh('records-journals')
+
+    yield
+
+    _delete_record('jou', 1936475)
```

Comment:
do you have to harcdode this? would be better to get this from the return value of `record_insert_or_replace`. Alternatively, @jacquerie suggests using real journals to be added to the demo records.

Reply:
But the problem is I do not know how to look for the journals that have the exact field that i need. It seems a bit time consuming? That is why I chose to mock them like this.

Reply:
IIRC, you found the journals you needed in inspirehep.net using the notation `CODE_FIELD:**`, right? If yes, we can proceed as the above comments were suggesting ;)